pkg/multus/multus.go

```diff
@@ -458,19 +458,34 @@ func delegateDel(exec invoke.Exec, pod *v1.Pod, ifName string, delegateConf *typ
 	return err
 }
 
+// CNICmd holds the required CNI spec data required to configure networking in
```

Comment:
CNICmd seems to be CNI command, 'ADD', 'DEL', 'CHECK'. How about to change it to CNIRuntimeArgs or MultusCNIArgs?

Reply:
I see your point. I can rename the struct accordingly.

I guess `MultusCNIArgs` makes more sense to me (in my own personal mental model).